lib/factory_girl/definition_proxy.rb

```diff
@@ -94,6 +95,16 @@ def transient(&block)
     #
     # are equivalent.
     def method_missing(name, *args, &block)
+      # If methods are called from debugger, original method should be called
+      if ['eval', 'process_commands', 'build_compact_name', 'build_compact_value_attr', 'print_element'].include?(caller_locations[1].base_label) ||
+          caller_locations[1].path == "(eval)" then
```

Comment:
Do not use `then` for multi-line `if`.<br>Align the operands of a condition in an `if` statement spanning multiple lines.